### PR TITLE
VACMS-16915: DUW Failure to Exhaust question

### PR DESCRIPTION
**Pull request by chriskim2311**
## Summary
This PR adds the last question Failure to Exhaust for DUW.

Note: Ignore the unit test file updates, I moved them into a questions folder. But added commented out E2E tests

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16915

## Testing done
**Route to question:**

Branch: Any
Discharge Year: Any
Reason:   Any BUT 'I received a DD215 that shows my discharge upgrade or correction. But I want an updated DD214.' OR 'I was discharged due to my sexual orientation (including under the Don’t Ask, Don’t Tell policy).'
Intention: 'No, I want to change only my characterization of discharge, re-enlistment code, separation code, or narrative reason for discharge.'
Court Martial: [No, my discharge was administrative or the outcome of a special or summary court-martial., I'm not sure]
Prev Application: “Yes”
Prev Application Year:  ['After 2014 ', 'After 2011',
 'After 2017']
Prev Application Type: [I applied to the Board for Correction of Naval Records (BCNR), I applied to a Board for Correction of Military Records (BCMR)]

Note: Cypress tests are purposely commented out since they will not work until we have the routes live on prod. These can be ran locally.

## Screenshots
Have_you_previously_applied_for_and_been_denied_a_discharge_upgrade_for_this_period_of_service____How_to_Apply_for_A_Discharge_Upgrade___Veterans_Affairs

## What areas of the site does it impact?
DUW

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

**Comment by laflannery** [comment]
Looks good!

File: src/applications/discharge-wizard/tests/v2/cypress/discharge-upgrade-wizard.cypress.spec.js [hunk 1]
```diff
@@ -52,6 +52,223 @@ xdescribe('Discharge Upgrade Wizard', () => {
 
       // PREVIOUS_APPLICATION
       h.verifyUrl(ROUTES.PREV_APPLICATION);
+      h.selectRadio(h.PREV_APPLICATION_INPUT, 0);
```
Comment (randimays) [inline src/applications/discharge-wizard/tests/v2/cypress/discharge-upgrade-wizard.cypress.spec.js:55]: Flagging (ignore if this is already done / irrelevant, etc.) that we'll have to test quite a few flows, and you may want to start breaking things down into multiple folders / files.

**Review by randimays — APPROVED** [review]
LGTM! Just left one non-blocking comment.

File: src/applications/discharge-wizard/tests/v2/cypress/discharge-upgrade-wizard.cypress.spec.js (hunk 1 above)
Comment (chriskim2311) [reply src/applications/discharge-wizard/tests/v2/cypress/discharge-upgrade-wizard.cypress.spec.js:55]: Thanks Randi! Yes I am definitely working on that, I will be testing all branched flows and skip ahead logic

**Pull request merged**